go.mod

```diff
@@ -126,7 +128,7 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/chrismellard/docker-credential-acr-env v0.0.0-20230304212654-82a0ddb27589 // indirect
 	github.com/clbanning/mxj/v2 v2.7.0 // indirect
-	github.com/cloudflare/circl v1.3.7 // indirect
+	github.com/cloudflare/circl v1.3.8 // indirect
```

Comment:
Do we really need the (I guess) unrelated dependency bumps like this to be included in this PR?

Reply:
These are brought in with executing `go mod tidy`.

Reply:
But why? Are they go-gather sub-dependencies?

Ideally this PR has just the minimum diff to main branch.